### PR TITLE
docs: Added the selector feature to docs

### PR DESCRIPTION
**Pull request by VenkatRamaraju**
Selectors help clients/users can identify a set of objects based on their labels.

**Description of the change:**
Added the selector feature to docs table with description, default, and link to Kubernetes API.

### CODE REVIEW

**Comment by tengqm** [comment]
Sounds to me that this needs a hold because the related feature PR #3275 is still under review.
/hold

**Comment by VenkatRamaraju** [comment]
@tengqm can the hold be removed now that PR #3275 has been merged?

File: website/content/en/docs/ansible/reference/watches.md [hunk 1]
```diff
@@ -93,8 +93,7 @@ Some features can be overridden per resource via an annotation on that CR. The o
 | Watching Cluster-Scoped Resources | `watchClusterScopedResources` | Allows the ansible operator to watch cluster-scoped resources that are created by ansible | | false | |
 | Max Runner Artifacts | `maxRunnerArtifacts` | Manages the number of [artifact directories](https://ansible-runner.readthedocs.io/en/latest/intro.html#runner-artifacts-directory-hierarchy) that ansible runner will keep in the operator container for each individual resource. | ansible.operator-sdk/max-runner-artifacts | 20 | |
 | Finalizer | `finalizer`  | Sets a finalizer on the CR and maps a deletion event to a playbook or role | | | [finalizers](../finalizers)|
-
-
+| Selector | `selector`  | Identifies a set of objects based on their labels | | None Applied | [Labels and Selectors] (https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)|
 #### Example
 ```YaML
 ---
```
Comment (camilamacedo86) [inline website/content/en/docs/ansible/reference/watches.md:99]: Could you add an example of selector in the doc? See that above has watches examples.

**Comment by VenkatRamaraju** [comment]
Made the changes.

File: website/content/en/docs/ansible/reference/watches.md [hunk 2]
```diff
@@ -110,5 +110,16 @@ Some features can be overridden per resource via an annotation on that CR. The o
...
+```YaML
+---
+- version: v1alpha1
+  group: test.example.com
+  kind: SelectorTest
+  playbook: playbooks/selector.yml
+  selector:
+    matchExpressions:
+     - {key: testLabel, operator: Exists, values: []}
+```
```
Comment (camilamacedo86) [inline website/content/en/docs/ansible/reference/watches.md:123]: IMO the example needs to be added under `An example Watches file:` with an explanation such as `# Simple example mapping Bar to a playbook with selectots`. Also, can you update the data of the example to more similar to the others (E.g bar foo and etc)?

**Pull request merged**